gcloud/datastore/connection.py

```diff
@@ -317,9 +304,9 @@ def commit(self, project, request_pb):
         request_pb.project_id = project
         try:
             return self._stub.Commit(request_pb, GRPC_TIMEOUT_SECONDS)
```

Comment:
You can stop using timeouts altogether. Of course it's up to you and your application's particular desired behavior to decide if that's the right thing to do.